lib/ansible/modules/source_control/github_release.py

```diff
@@ -77,13 +93,15 @@
     HAS_GITHUB_API = True
 except ImportError:
     HAS_GITHUB_API = False
+from ansible.module_utils.basic import AnsibleModule, get_exception
 
 
 def main():
     module = AnsibleModule(
         argument_spec=dict(
             repo=dict(required=True),
             user=dict(required=True),
+            password=dict(required=True),
```

Comment:
Is this actually a required option?

1) Are `password` and `token` mutually exclusive. 
2) can you use this without auth? 


There are some options to `AnsibleModule` which you may be able to use to validate options passed in.
Look at existing modules for examples:
```
mutually_exclusive
required_one_of
```